src/main/scala/temple/builder/project/ProjectBuilder.scala

```diff
@@ -35,6 +37,16 @@ object ProjectBuilder {
         (File(s"${name.toLowerCase}", "Dockerfile"), dockerfileContents)
     }
 
-    Project(databaseCreationScripts ++ dockerfiles)
+    val kubeFiles = KubernetesGenerator.generate(
+      KubernetesBuilder
+        .createServiceKubeFiles(
```

Comment:
Please could this go on the same line as `KubernetesBuilder`, the chaining doesn’t make sense if there isn’t a flow of information (KubernetesBuilder isn’t some data to be passed down the pipeline)